.github/terraform/integration_databricks/data_and_varaibles.tf

```diff
@@ -0,0 +1,44 @@
+data "azurerm_client_config" "current" {}
+
+data "azurerm_subscription" "primary" {}
+
+data "azuread_service_principal" "captain" {
+  display_name = module.config.integration.captain.display_name
+}
+
+data "azurerm_databricks_access_connector" "ext_access_connector" {
+  name                = module.config.integration.resource_name
+  resource_group_name = module.config.integration.rg_name
+}
+
+data "azurerm_key_vault" "key_vault" {
+  name                = module.config.integration.resource_name
+  resource_group_name = module.config.integration.rg_name
+}
+
+data "azurerm_key_vault_secret" "captain_spn_id" {
+  name         = module.config.integration.captain.kv_secret_id
+  key_vault_id = data.azurerm_key_vault.key_vault.id
+}
+
+data "azurerm_key_vault_secret" "captain_spn_password" {
+  name         = module.config.integration.captain.kv_secret_pass
+  key_vault_id = data.azurerm_key_vault.key_vault.id
+}
+
+data "azurerm_key_vault_secret" "captain_spn_tenant" {
+  name         = module.config.integration.captain.kv_secret_tenant
+  key_vault_id = data.azurerm_key_vault.key_vault.id
+}
+
+data "databricks_metastore" "db_metastore" {
+  provider = databricks.account
+
```

Comment:
remove space

Reply:
Not sure which space you mean. I use `terraform fmt` so I will not change spacing inside lines. I have consistently separated the provider with a newline everywhere, so I also want to keep that.